src/components/scenes/RequestScene.tsx

```diff
@@ -572,7 +519,9 @@ const getStyles = cacheStyles((theme: Theme) => ({
   },
   accessoryButton: {
     paddingVertical: theme.rem(0.5),
-    paddingHorizontal: theme.rem(1)
+    paddingHorizontal: theme.rem(1),
+    borderColor: 'red',
+    borderWidth: 0.5
   },
```

Comment:
This whole style can be deleted now? Maybe also `styles.accessoryView`? And according to my searching, `theme.inputAccessoryBackground` will also be unused after this.